saleor/dashboard/product/forms.py

```diff
@@ -2,77 +2,161 @@
 
 from django import forms
 from django.forms.models import inlineformset_factory
-from django.forms.widgets import ClearableFileInput
+from django.utils.text import slugify
+from django.utils.translation import pgettext_lazy
 
-from ...product.models import (ProductImage, Product, ShirtVariant, BagVariant,
-                               Shirt, Bag)
+from django_prices.templatetags.prices_i18n import gross
+
+from ...product.models import (ProductImage, Stock, ProductVariant, Product,
+                               ProductAttribute, AttributeChoiceValue)
 
 PRODUCT_CLASSES = {
-    'shirt': Shirt,
-    'bag': Bag}
+    Product: 'Default'
+}
 
 
-class ProductClassForm(forms.Form):
-    cls_name = forms.ChoiceField(
-        choices=[(name, name.capitalize()) for name in PRODUCT_CLASSES.keys()])
+def get_verbose_name(model):
+    return model._meta.verbose_name
 
 
-class ProductForm(forms.ModelForm):
-    class Meta:
-        model = Product
-        fields = ['name', 'description', 'collection']
+class ProductClassForm(forms.Form):
+    product_cls = forms.ChoiceField(
+        label=pgettext_lazy('Product class form label', 'Product class'),
+        widget=forms.RadioSelect,
+        choices=[(cls.__name__, presentation) for cls, presentation in
+                 PRODUCT_CLASSES.iteritems()])
+
+    def __init__(self, *args, **kwargs):
+        super(ProductClassForm, self).__init__(*args, **kwargs)
+        self.fields['product_cls'].initial = PRODUCT_CLASSES.keys()[0].__name__
 
 
-class ShirtForm(ProductForm):
+class StockForm(forms.ModelForm):
     class Meta:
-        model = Shirt
+        model = Stock
         exclude = []
+        widgets = {
+            'product': forms.HiddenInput()
```

Comment:
Why is there a `product` field in the form? What if I intentionally alter the hidden value?